rrfs-test/validated_yamls/templates/obtype_config/abi_g16.yaml

```diff
@@ -0,0 +1,488 @@
+           minvalue: 0.99
+
+       - filter: RejectList
+         filter variables:
+         - name: brightnessTemperature
+           channels: *abi_g16_channels
+         where:
+         - variable:
+             name: GeoVaLs/land_area_fraction
+           maxvalue: 0.99
+           max_exclusive: true
+         - variable:
+             name: GeoVaLs/water_area_fraction
+           maxvalue: 0.99
+           max_exclusive: true
+         - variable:
+             name: GeoVaLs/ice_area_fraction
+           maxvalue: 0.99
+           max_exclusive: true
+         - variable:
+             name: GeoVaLs/surface_snow_area_fraction
+           maxvalue: 0.99
+           max_exclusive: true
+
+       # BT range check
+       - filter: Bounds Check
+         filter variables:
+         - name: brightnessTemperature
+           channels: *abi_g16_channels
+         minvalue: 0.0
+         maxvalue: 1000.0
+         action:
+           name: reject
+
+       # Model top transmittance check
+       - filter: Perform Action
+         filter variables:
+         - name: brightnessTemperature
+           channels: *abi_g16_channels
+         action:
+           name: inflate error
+           inflation variable:
+             name: ObsFunction/ObsErrorFactorTransmitTopRad
+             channels: *abi_g16_channels
+             options:
+               channels: *abi_g16_channels
+   
+       # Cloud detection
+       - filter: Bounds Check
+         filter variables:
+         - name: brightnessTemperature
+           channels: *abi_g16_channels
+         test variables:
+         - name: ObsFunction/CloudDetectMinResidualIR
+           channels: *abi_g16_channels
+           options:
+             channels: *abi_g16_channels
+             use_flag: [-1, 1, 1, 1, -1, -1, -1, -1, -1, -1]
+             use_flag_clddet: [-2, -2, -2, -2, -2, -2, -2, -2, -2, -2]
+             obserr_dtempf: [0.5, 2.0, 4.0, 2.0, 4.0]        
+             error parameter vector: [ 2.2, 1.756, 1.587, 1.185, 2.2, 2.2, 2.2, 2.2, 2.2, 2.2 ]
+         maxvalue: 1.0e-12
+         action:
+           name: reject
+
+       # Scene Consistency Check based on ch13 (10.8 micron)
+       - filter: Domain Check
+         filter variables:
+         - name: brightnessTemperature
+           channels: 7, 10-16
+         where:
+         - variable:
+             name: ClearSkyStdDev/brightnessTemperature
+             channels: 13
+           maxvalue: 0.5
+           max_exclusive: true
+
+       # Scene Consistency Channels 8-10 Based on ch 8 SDTB
+       - filter: Perform Action
+         filter variables:
+         - name: brightnessTemperature
+           channels: 8-10
+         where:
+         - variable:
+             name: ClearSkyStdDev/brightnessTemperature_8
+           maxvalue: 0.5
+           minvalue: 0.4
+           min_exclusive: true
+         action:
+           name: inflate error
+           inflation factor: 1.1489
+
+       - filter: Perform Action
```

Comment:
Please add a comment here.

Reply:
> Please add a comment here.

Modified the comments for easier to understand. But jut leave one comments for this filter because they apply the the different inflate factor based on the difference stander deviation range.